### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by Mhassanbughio**
@EddieHubCommunity/awesome-github-profiles please checkout profile.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1548,3 +1548,10 @@
     "issueNumber": 805
   }
 ]
+
+{
```
Comment (CodingSpecies) [inline data.json:1552]: Please can you move you addition to under line `1549` please @Mhassanbughio

File: data.json [hunk 2]
```diff
@@ -1548,3 +1548,10 @@
...
+    "name": "Muhammmad Hassan",
+    "githubUsername": "Mhassanbughio",
+    "imageUrl": "https://user-images.githubusercontent.com/60064591/139943235-68ea4d39-9962-46fa-b6f9-584385f77a63.jpg",
+    "issueNumber": 819
+  },
```
Comment (CodingSpecies) [inline data.json:1557]: Please can you remove this comma `,`

File: data.json [hunk 3]
```diff
@@ -1548,3 +1548,10 @@
     "issueNumber": 805
   }
 ]
+
+{
+    "name": "Muhammmad Hassan",
+    "githubUsername": "Mhassanbughio",
+    "imageUrl": "https://user-images.githubusercontent.com/60064591/139943235-68ea4d39-9962-46fa-b6f9-584385f77a63.jpg",
+    "issueNumber": 819
```
Comment (CodingSpecies) [inline data.json:1556]: I think this issue number should be `818` @Mhassanbughio

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Comment by Panquesito7** [comment]
Duplicate of #819.

**Pull request closed without merging**